src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.styles.js

```diff
@@ -1,5 +1,17 @@
-import styled, { css } from 'styled-components';
+import styled, { css, keyframes } from 'styled-components';
 import colors from '../../styles/colors';
+import fonts from '../../styles/fonts';
+import spacings from '../../styles/spacings';
+
+const progressBar = keyframes`
```

Comment:
Wha is `keyframes`? do you have a link with a simple introduction?

Reply:
Keyframes is to make animations
https://www.styled-components.com/docs/basics#animations

Reply:
Nice! thanks!